### ISSUE TITLE
Accessibility Overview

### ISSUE THREAD

**Issue by hannahackermans** [smbjorklund/cellproject#57]
I have tested CELL for Web Content Accessibility Guidelines (WCAG) 2.1 International Standard according to the requirements of uutilsynet.no, which covers all A and AA success criteria, but not AAA. Some requirements are more intricate than others and I am still new to this, so there might be some things that I’ve overlooked that I can address in the next report.

The table gives an overview of the current state of the accessibility of CELL. I will turn the failures into GitHub issues labeled with "accessibility" and I will update this table to include the right issues. I am leaving some issues behind at this point because some issues overlap with parts of the website that might be overhauled anyway. We can revisit those issues when we have a better sense of the goals and tasks going forward. 

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/hac061/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/hac061/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#1155CC" vlink="#1155CC">

WCAG Requirement | Current status
-- | --
1.1.1 Non-text Content | Fail, issue #58 
  |  
1.2.1 Audio-only and Video-only   (Prerecorded) | not applicable
1.2.2 Captions (Prerecorded) | not applicable
  |  
1.3.1 Info and Relationships | Fail, issue #59 and #60 
1.3.2 Meaningful Sequence | Fail, issue #61 
1.3.3 Sensory Characteristics | Fail, issue #62 
1.3.4 Orientation | Pass
1.3.5 Identify Input Purpose | not applicable
  |  
1.4.1 Use of Color | Fail, issue #62 
1.4.2 Audio Control | not applicable
1.4.3 Contrast (Minimum) | Fail, issue #62
1.4.4 Resize text | Pass
1.4.5 Images of Text | Fail, issue #58 
1.4.10 Reflow | Pass
1.4.11 Non-text Contrast | Fail, issue #62 
1.4.12 Text Spacing | Pass
1.4.13 Content on Hover or Focus | not applicable
  |  
2.1.1 Keyboard | Fail, issue #60 
2.1.2 No Keyboard Trap | Pass
2.1.4 Character Key Shortcuts | not applicable
  |  
2.2.1 Timing Adjustable | not applicable
2.2.2 Pause, Stop, Hide | not applicable
  |  
2.3.1 Three Flashes or Below Threshold | not applicable
  |  
2.4.1 Bypass Blocks | Fail, issue #64 
2.4.2 Page Titled | Fail, issue #65 
2.4.3 Focus Order | Pass
2.4.4 Link Purpose (In Context) | Fail, issue #65 
2.4.5 Multiple Ways | Pass
2.4.6 Headings and Labels | Pass
2.4.7 Focus Visible | Fail, issue #63 
  |  
2.5.1 Pointer Gestures | not applicable
2.5.2 Pointer Cancellation | Pass
2.5.3 Label in Name | Fail, issue ###
2.5.4 Motion Actuation | not applicable
  |  
3.1.1 Language of Page | Fail, issue #66 
3.1.2 Language of Parts | Fail, issue #66 
  |  
3.2.1 On Focus | Pass
3.2.2 On Input | Pass
3.2.3 Consistent Navigation | Pass
3.2.4 Consistent Identification | Fail, issue #65 
  |  
3.3.1 Error Identification | not applicable
3.3.2 Labels or Instructions | Fail, issue ###
3.3.3 Error Suggestion | not applicable
3.3.4 Error Prevention (Legal,   Financial, Data) | not applicable
  |  
4.1.1 Parsing | marked as fail but need to discuss with @steinmb
4.1.2 Name, Role, Value | marked as fail but need to discuss with @steinmb
4.1.3 Status Messages | Fail, issue ###

</body>

</html>
